task7.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
+
+if __name__ == "__main__":
+    arr = np.random.rand(4, 4)
```

Comment:
How can you turn this into a reusable function?
Without documenteation how can a reader know what you're trying to do?